classes/services/queryBuilders/PKPSubmissionQueryBuilder.inc.php

```diff
@@ -284,7 +284,7 @@ public function getQuery() {
 					$join->on('publication_tlps.locale', '=', Capsule::raw("'{$locale}'"));
 				});
 			$q->leftJoin('publications as publication_tlpl', 's.current_publication_id', '=', 'publication_tlpl.publication_id')
-				->leftJoin('publication_settings as publication_tlpsl', function (object $join){
+				->leftJoin('publication_settings as publication_tlpsl', function (object $join) {
```

Comment:
@AlineLepidus The type should be `JoinClause` (`use Illuminate\Database\Query\JoinClause;`), also we're normally writing it like:
```php
$join->on(...)
  ->on(...)
  ->on(...);
```

If you can update, great! If not, no problem :)

Reply:
As I mentioned above, you should be able to use `$join->where(...)` instead of `$join->on` when using literal constraints, instead of needing to wrap the literal in `Capsule::raw`.